index.html

```diff
@@ -4352,17 +4352,13 @@ <h3>Parsing content</h3>
 <section> <h2 id="security">Security and Privacy</h2>
   <section> <h3>Chain of trust</h3>
     <p>
-      Web sites and applications using Web NFC are not trusted.
-      This means that the user needs to be made aware of what NFC functionality
-      a web page intends to do. Implementations need to
-      make sure that when the user authorizes a method of this API, only
-      that action is run, without side effects, and exactly in the context and
-      the number of times the user allows the execution of NFC related
-      operations, according to the algorithmic steps detailed in this
-      specification.
```

Comment:
I would keep this, as it provides clarity for implementations. @ereshetova ?

Reply:
Some of it was added elsewhere... and it is implied that you need to implement the algorithms as described. I also find it hard to understand

Reply:
OK, do as you wish :) 
Though this sentence makes sense:
"Implementations need to make sure that when the user authorizes a method of this API, only that action is run, without side effects"